src/main/java/seedu/exercise/logic/commands/ScheduleCommand.java

```diff
@@ -22,4 +22,9 @@
             + PREFIX_INDEX + "INDEX OF SCHEDULE\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_INDEX + "1";
+
+    @Override
+    public String getCommandWord() {
+        return COMMAND_WORD;
```

Comment:
Also, why not have only `UndoableCommand` implement this method since not every Command needs to use this method

Reply:
Good point. Rectified